src/components/BrowseResultsList/getBrowseResultsFormatter.test.js

```diff
@@ -171,6 +178,9 @@ describe('getBrowseResultsFormatter', () => {
       // Default row
       expect(screen.getByText(nonAnchorRecord.name).tagName.toLowerCase()).not.toBe('strong');
       expect(screen.getByText(nonAnchorRecord.totalRecords).tagName.toLowerCase()).not.toBe('strong');
+      // Non clickable row
+      expect(screen.getByText(notClickableRecord.name).tagName.toLowerCase()).not.toBe('strong');
```

Comment:
`strong` tag doesn't mean that the element is not clickable, it's better to check that the element doesn't have `href` attribute for example.

Reply:
This tests just checks that this item was rendered. There is a second test that checks that the item is not clickable and does not redirect to another page

Reply:
Then why do you need to check that tag name is not strong? To check that the item was rendered will be enough to use `toBeInTheDocument()`. I don't quite understand this `strong` checking, is this header or something?

Reply:
It shouldn't be `strong` because it's not a full match result and doesn't have any associated records